Swift/QtUI/QtXMLSenderWidget.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2017 Barun Parruck.
+ * All rights reserved.
+ * See the COPYING file for more information.
+ */
```

Comment:
Your copyright header should match the license you aim to submit the patch under. See for example https://github.com/swift/swift/commit/cc873b3f00db4cd0a778bc2ec04f8748d70a92f9#diff-0856e1d938fb0b1106ce0267bbb319b0R1 .